server/startup/migrations/v243.ts

```diff
@@ -0,0 +1,43 @@
+import { addMigration } from '../../lib/migrations';
+import { Uploads } from '../../../app/models/server';
+
+
+addMigration({
+	version: 243,
+	up() {
+		const images = Uploads.model.rawCollection().find({ typeGroup: 'image' });
+		const getAspectRatio = (image: Record<string, any>): number => image.identify.size.width / image.identify.size.height;
```

Comment:
trying with an old database it found a some images without `identify` field, which broke the migration